backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -2466,13 +2468,12 @@ void SurfaceSdlGraphicsManager::handleScalerHotkeys(int scalefactor, int scalerT
 			g++;
 		}
 		if (newScalerName) {
-			const Common::String message = Common::String::format(
-				"%s %s\n%d x %d -> %d x %d",
-				_("Active graphics filter:"),
+			const Common::U32String message = Common::U32String::format(
+				_("Active graphics filter:") + Common::U32String("%s\n%d x %d -> %d x %d"),
```

Comment:
Why are you constructing the format string like this? Please revert to the original code.